pgjdbc/src/main/java/org/postgresql/Driver.java

```diff
@@ -659,7 +659,7 @@ private static long timeout(Properties props) {
       try {
         return (long) (Float.parseFloat(timeout) * 1000);
       } catch (NumberFormatException e) {
-        LOGGER.log(Level.WARNING, "Couldn't parse loginTimeout value: {0}", timeout);
+        LOGGER.log(Level.FINE, "Couldn't parse loginTimeout value: {0}", timeout);
```

Comment:
-1 as this happens in case user provides loginTimeout property and it gets ignored.
Does that really happen often?
I think this log message would appear just in case one tries to put invalid value, and it would warn that the value is ignored.